artman/conductors/cloudtasks_conductor.py

```diff
@@ -114,14 +153,16 @@ def _prepare_dir(source_repo="https://github.com/googleapis/googleapis.git"):
     TODO(ethanbao): support loading more input files from heterogeneous data
     sources"""
 
-    repo_root = '/tmp/artman/%s' % str(uuid.uuid4())
+    task_id = str(uuid.uuid4())
+    repo_root = '/tmp/artman/%s' % task_id
     logger.info('Prepare a temporary root repo: %s' % repo_root)
     try:
         os.makedirs(repo_root)
     except OSError as e:
         raise e
     logger.info('Checking out fresh clone of %s.' % source_repo)
     googleapis_dir = os.path.join(repo_root, "googleapis")
+    subprocess.check_output(['rm', '-f', '.git/config'])
```

Comment:
TODO: We should move `TaskBase.exec_command` into a general utility function and use it instead of `subprocess.check_output`.